### ISSUE TITLE
Node version was not changed in the installer

### ISSUE THREAD

**Issue by mh-concordium** [Concordium/concordium-node#380]
**Bug Description**
Installing node 4.1.0 looks like node is still 4.0.11

**Steps to Reproduce**
Install 4.1.0 Node on Windows.

**Expected Result**
Version in the installer and application properties should be 4.1.0.

**Actual Result**
Version in the installer and application properties is 4.0.11

**Versions**
- Software Version: 4.1.0
- OS: Windows

![image](https://user-images.githubusercontent.com/98821384/172612930-f25cb0ec-bc03-4683-8934-d1e7cb920dd9.png)

![image](https://user-images.githubusercontent.com/98821384/172612987-053e4cfd-7544-4d02-bc97-6028f9b766c1.png)

**Comment by abizjak** [comment]
Just to confirm, the statement
```
Actual Result
Version in the installer and application properties is 4.0.7
```
has a typo, correct? 4.0.7 should be 4.0.11?

**Comment by abizjak** [comment]
Closed by #387
